torchaudio/_soundfile_backend.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+import torch
+
+
+def check_input(src):
+    if not torch.is_tensor(src):
+        raise TypeError("Expected a tensor, got %s" % type(src))
+    if src.is_cuda:
+        raise TypeError("Expected a CPU based tensor, got %s" % type(src))
+
+
+def load(
+    filepath,
+    normalization=True,
+    channels_first=True,
+    num_frames=0,
+    offset=0,
+    filetype=None,
+    **_
```

Comment:
Other parameters will be silently dropped. This may not be desired, but it convenient in code :)

Reply:
I removed this.